rdt-open-reader/src/main/java/io/ona/rdt/util/RDTJsonFormUtils.java

```diff
@@ -239,6 +239,7 @@ public void startJsonForm(JSONObject form, Activity context, int requestCode) {
         intent.putExtra(JSON_FORM_PARAM_JSON, form.toString());
         intent.putExtra(PERFORM_FORM_TRANSLATION, true);
         context.startActivityForResult(intent, requestCode);
+        io.ona.rdt.util.Utils.verifyUserAuthorization(context);
```

Comment:
Isn't this what we were avoiding last time? This will be called every time a form is launched?

Reply:
https://github.com/onaio/rdt-standard/issues/958#issuecomment-821310139

Reply:
https://github.com/onaio/rdt-standard/issues/958#issuecomment-822229869

Reply:
feedback implemented